index.bs

```diff
@@ -1660,6 +1334,94 @@ multiple transactions per microframe. In a SuperSpeed device this value will
 include the multiplication factor provided by the <code>bMaxBurst</code> field
 of the SuperSpeed Endpoint Companion descriptor.
 
+# Integrations # {#integrations}
+
+## Feature Policy ## {#feature-policy}
+
+This specification defines a <a>feature</a> that controls whether the
+{{Navigator/usb}} attribute is exposed on the {{Navigator}} object.
```

Comment:
Since WebUSB is a new API I would prefer to restrict attribute exposure unless that isn't considered to be a good practice.

The argument against restricting attribute exposure is that it makes it harder for developers to discover why code is not working because we don't have the opportunity to throw a SecurityError with a helpful message.